### DISCUSSION TITLE
Mode question

### DISCUSSION THREAD

**Discussion by mosheng-eng** [Cyfrin/foundry-full-course-cu#2922] [General]
![image](https://github.com/user-attachments/assets/6f6f1f5d-2da1-4a33-8873-d2056573cd18)

this  is a snapshot from course "Foundry Smart Contract Lottery". The mode result is not right.

**Comment by EngrPips** [comment]
Hello @mosheng-eng, Thanks for pointing this out. Please report this in the `issue` tab of this GitHub.
